src/hotspot/share/opto/c2_globals.hpp

```diff
@@ -679,9 +679,11 @@
                                                                             \
   develop(uint, VerifyIterativeGVN, 0,                                      \
           "Verify Iterative Global Value Numbering"                         \
-          "=XY, with Y: verify Def-Use modifications during IGVN"           \
-          "          X: verify that type(n) == n->Value() after IGVN"       \
-          "X and Y in 0=off; 1=on")                                         \
+          "=DCBA, with A: verify Def-Use modifications during IGVN"         \
+          "            B: verify that type(n) == n->Value() after IGVN"     \
+          "            C: verify all Node::Ideal were applied in IGVN"      \
+          "            D: verify all Node::Identity were applied in IGVN"   \
```

Comment:
```suggestion
          "            C: verify Node::Ideal did not miss opportunities"    \
          "            D: verify Node::Identity did not miss opportunities" \
```
For me, "all `Node::Ideal` were applied" parsed weirdly, so I tried my hand at an alternative formulation. Feel free to ignore.